docs/FAQ.md

```diff
@@ -0,0 +1,98 @@
+* [How do I force single threaded execution mode in ORT?](#How-do-I-force-single-threaded-execution-mode-in-ORT)
+
+**Section 2: [Troubleshooting](#troubleshooting)**
+* [Performance Troubleshooting Checklist](#performance-troubleshooting-checklist)
+* [Why is my model running slower on GPU than CPU?](#Why-is-my-model-running-slower-on-GPU-than-CPU)
+* [My converted Tensorflow model is slow - why?](#My-converted-Tensorflow-model-is-slow---why)
+* [My PyTorch model is running slower on GPU - why?](#My-PyTorch-model-is-running-slower-on-GPU---why)
+* [I can't build the xyz execution provider.](#I-cant-build-the-xyz-execution-provider)
+
+***
+# FAQ 
+
+## What type of GPU is supported? 
+The default ONNX Runtime GPU execution provider is for NVIDIA GPUs with CUDA. This build is published on Nuget: [Microsoft.ML.Onnxruntime.Gpu](https://www.nuget.org/packages/Microsoft.ML.OnnxRuntime.gpu) and PyPi: [onnxruntime-gpu](https://pypi.org/project/onnxruntime-gpu). To build from source, see [BUILD.md](https://github.com/microsoft/onnxruntime/blob/master/BUILD.md#cuda). 
+
+Additionally, for NVIDIA GPUs you can build the [TensorRT execution provider](./execution_providers/TensorRT-ExecutionProvider.md) following [these instructions from BUILD.md](./../BUILD.md#tensorrt)
+
+If your deployment target is Windows, we recommend using the [DirectML execution provider](./execution_providers/DirectML-ExecutionProvider.md) for optimal performance and compatibility with a broad set of GPUs.  See the build instructions [here](./../BUILD.md#directml).
+
+AMD GPU support is in progress.
+
+## Which Execution Provider will give me the best performance? 
+Performance is dependent on the specific model you're trying to run, the session and run options you've selected, and of course, your specific hardware target. Please refer to the [Performance Tuning](./ONNX_Runtime_Perf_Tuning.md) page for guidance on how to performance tune ONNX Runtime for your model. You can use the [ONNX Go Live - OLive](https://github.com/microsoft/OLive) tool to help test and identify the best configuration and execution provider to use.  Below you'll find some more information that may be helpful to select the right Execution Provider.
+
+### CUDA (Default GPU) vs CPU?
+The CPU version of ONNX Runtime provides a complete implementation of all operators in the ONNX spec. This ensures that your ONNX-compliant model can execute successfully (*minor caveat: not 100% of types are supported to minimize the binary size, but can usually be easily added as needed*). On the other hand, not all CUDA kernels are implemented, as these have been prioritized on an as-needed basis. As a result, this means that if your model contains operators that do not have a CUDA implementation, it will fall back to CPU. Switching between CPU and GPU can cause significant performance impact.
```

Comment:
"easily added as needed" - can the user really do this by themselves? if so, need to link to docs.
the wording should be updated here - "In order to keep the binary size small, common data types are supported for the ops. If you are using an uncommon data type, <do this>" and similarly for the CUDA sentences